config/test.js

```diff
@@ -5,8 +5,8 @@ module.exports = {
 		name: 'hah-test'
 	},
 	facebook: {
-		appId: process.env.FACEBOOK_APP_ID,
-		appSecret: process.env.FACEBOOK_APP_SECRET,
+		appId: process.env.V_FACEBOOK_APP_ID,
+		appSecret: process.env.V_FACEBOOK_APP_SECRET,
```

Comment:
Ah ok. We may still need one in the future for people who want to sign in and add their local resources. What do you think about making it more descriptive though `VERIFICATION_PORTAL_FACEBOOK_APP_ID` or something like that? I know that's long, but I'm definitely a fan of verbose/descriptive names over short names. Thoughts?